web/modules/custom/dpl_event/src/PriceFormatter.php

```diff
@@ -15,11 +17,44 @@
 class PriceFormatter {
 
   /**
-   * Constructor.
+   * The currency that prices should be shown in (ISO 4217 code).
+   */
+  public string $currency = 'DKK';
+
+  /**
+   * A possible prefix, shown in currency labels. E.g. "€ ".
+   */
+  public ?string $currencyPrefix = NULL;
+
+  /**
+   * A possible suffix, shown in currency labels. E.g. " DKK".
+   */
+  public ?string $currencySuffix = NULL;
+
+  /**
+   * {@inheritDoc}
    */
   public function __construct(
     protected TranslationInterface $translation,
-  ) {}
+    protected ConfigFactoryInterface $configFactory,
+  ) {
+    $config = $this->configFactory->get(SettingsForm::CONFIG_NAME);
+
+    $currency = $config->get('price_currency') ?? $this->currency;
+    $this->currency = $currency;
+
+    switch ($currency) {
+      case 'DKK':
+        $this->currencyPrefix = NULL;
```

Comment:
The property is already initiated as NULL. I do not see any reason to setting the value again.